gunicorn/workers/ggeventpause.py

```diff
@@ -11,22 +11,25 @@
 extra operational niceties.
 """
 
+import logging
```

Comment:
Can you switch to using the Lyft logging library? We'll end up with events that can automatically be parsed by the logging pipeline.